src/main.js

```diff
@@ -127,6 +135,68 @@ class MiniGraphCard extends LitElement {
     }
   }
 
+  checkSections() {
+    let root = document.querySelector('home-assistant');
+    root = root && root.shadowRoot;
+    root = root && root.querySelector('home-assistant-main');
+    root = root && root.shadowRoot;
+    root = root && root.querySelector('ha-drawer');
+    root = root && root.querySelector('partial-panel-resolver');
+    root = root && root.shadowRoot || root;
+    root = root && root.querySelector('ha-panel-lovelace');
+    root = root && root.shadowRoot;
+    root = root && root.querySelector('hui-root');
+    if (root) {
+      const ll = root.lovelace;
+      ll.current_view = root.___curView;
+      const panelType = ll.rawConfig.views[ll.current_view].type;
+      return panelType === 'sections';
+    }
+    return false;
+  }
+
+  getCurrentLayout() {
```

Comment:
I wonder: isn't this taken care of by home assistant? If we report a minimum row count via `getLayoutOptions`, shouldn't this be reflected in `this.config.layout_options`?

See also here, maybe: https://github.com/home-assistant/frontend/blob/67217b9dd0f5ad43aafb3bba2fe2821407b25243/src/panels/lovelace/editor/card-editor/hui-card-layout-editor.ts#L68

Reply:
No, it only clamps the values for setting the card size and for the visual editor, values set by user in `this.config.layout_options` stays untouched even when they're lower than minimum set in `getLayoutOptions`.